wallet/src/Ergvein/Wallet/Page/Receive.hs

```diff
@@ -60,7 +61,7 @@ receivePageWidget cur i EgvPubKeyBox{..} = do
     setFlagToExtPubKey "receivePageWidget:1" $ (cur, i) <$ newE
     clipboardCopy (keyTxt <$ copyE)
     divClass "receive-adr-andr" $ text $ "#" <> showt i <> ": " <> keyView
-    labelD <- divClass "button-receive" $ textFieldNoLabel $ getLabelFromEgvPubKey pubKeyBox'key
+    labelD <- divClass "button-receive" $ textInput (getLabelFromEgvPubKey pubKeyBox'key) M.empty never never
```

Comment:
Consider refactor arguments to structure that will allow adding default named arguments.

Like `def & label .~ etLabelFromEgvPubKey pubKeyBox'key`